src/domain/occurrence/__tests__/CreateOccurrencePage.test.tsx

```diff
@@ -130,10 +129,20 @@ afterEach(() => {
   jest.restoreAllMocks();
 });
 
-test('renders coming occurrences table correctly', async () => {
-  initializeMocks();
-  renderWithRoute(<CreateOccurrencePage />, {
+const InitializeMocksAndRenderPage = (
+  renderOptions,
+  fromDate = new Date(2020, 7, 2),
+  occurences = 5
+) => {
+  initializeMocks(fromDate, occurences);
```

Comment:
After this refactor there is no longer need for mocks to be initialized to global scope of this file. They could be return form this function now. I think that approach would be easier to understand.

Reply:
fixed